packages/components/src/Components/FilterChips/filter-chips.scss

```diff
@@ -0,0 +1,9 @@
+.ins-c-chip-group__plain > li {
+    background: none;
+    padding: 0;
+    background-color: initial !important;
```

Comment:
Can we make this parent more specific to eliminate !important

Reply:
Good point. I was frustrated that PF rewrite these styles, well I went with more specific rule and it works, thanks!